content/courses/ada_2022_whats_new/chapters/array_aggregates.rst

```diff
@@ -0,0 +1,98 @@
+.. include:: ../../global.txt
+
+Advanced Array Aggregates
+=========================
+
+.. note::
+
+    Such array aggregates are supported by
+
+    * GNAT Community Edition 2020
+    * GCC 11
+
+Square brackets
+---------------
+
+In Ada 2022 you can use square brackets in array aggregates. Square
+brackets usage simplifies writing of an empty aggregate and a single
+element aggregate. Consider this:
+
+.. code-block:: ada
```

Comment:
The tests indicate an error here:

```
array_aggregates.rst:20:Could not lex literal_block as "ada". Highlighting skipped.
```

You might need to deactivate syntax highlighting for this code block. Alternatively, you could try and use the widget.

Reply:
Using a widget works fine in this specific case. I've just tried this out locally:

```
.. code:: ada compile_button manual_chop project=Courses.Ada_2022_Whats_New.Show_Square_Brackets

   !show_square_brackets.ads
   pragma Ada_2022;
   pragma Extensions_Allowed (On);

   package Show_Square_Brackets is
```

Reply:
I guess you could use this approach for the remaining blocks that are currently failing. I see these warnings locally:

```
/vagrant/content/courses/ada_2022_whats_new/chapters/assignment_target.rst:37: WARNING: Could not lex literal_block as "ada". Highlighting skipped.
/vagrant/content/courses/ada_2022_whats_new/chapters/assignment_target.rst:58: WARNING: Could not lex literal_block as "ada". Highlighting skipped.
/vagrant/content/courses/ada_2022_whats_new/chapters/image_attribute.rst:67: WARNING: Could not lex literal_block as "ada". Highlighting skipped.
```